File: run_explore.js
const { execSync } = require('child_process');
try {
  const result = execSync('node src/web/scripts_B/explore_page.js', { 
    cwd: '/', 
    encoding: 'utf-8', 
    timeout: 60000,
    maxBuffer: 10 * 1024 * 1024
  });
  console.log(result);
} catch (err) {
  console.error('STDOUT:', err.stdout);
  console.error('STDERR:', err.stderr);
  console.error('Error:', err.message);
}

File: run_explore.py
import subprocess
result = subprocess.run(['node', 'src/web/scripts_B/explore_page.js'], cwd='/', capture_output=True, text=True, timeout=60)
print("=== STDOUT ===")
print(result.stdout)
print("=== STDERR ===")
print(result.stderr)
print("=== RETURN CODE ===")
print(result.returncode)
